cmds/subscribe.c

```diff
@@ -31,10 +34,12 @@ static bool subscription_generator(
     }
 
     if (!w_query_process_file(query, ctx, f)) {
+      *num_walked = n;
```

Comment:
People are definitely going to forget doing this if they add return paths in the future. Can you add:

* a `done` label
* a bool `ret` value which can be set to `false` or `true`
* switch these internal returns to `goto done`?